ex4/server/middleware/logger.js

```diff
@@ -0,0 +1,7 @@
+export function logger(req, res, next) {
```

Comment:
cool. worth changing the name to `requestLogger` as logger is more generic name and the functionality is coupled to incoming requests